projects/storefrontlib/shared/services/file/files-form-validators.ts

```diff
@@ -11,6 +11,9 @@ import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
   providedIn: 'root',
 })
 export class FilesFormValidators {
+  multiplier: number = 1000000;
```

Comment:
```suggestion
  TO_MB = 1000000;
  or
  ONE_MB  = 1000000
```
It should be 10 MBs, right?   If you don't know the number leave it as it was

Reply:
Yes, that is right